docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
       interval: 3s
       timeout: 5s
       retries: 3
-    entrypoint: '.ci/entrypoint.sh'
+    entrypoint: bigchaindb start
```

Comment:
`.ci/entrypoint.sh` is need for other travis matrix tests which use ABCI proxy app as fixture. Keep `entrypoint: '.ci/entrypoint.sh` should work too.